src/lib/bridge/Bridge.ts

```diff
@@ -106,4 +145,9 @@ export default class Bridge {
             window.opener.postMessage(message, '*');
         }
     }
+
+    public sendPaymentRequest(receiver: string, amount: number, assetType: string, issuer: string) {
```

Comment:
I think this function has too many parameters, can you group them into a type or interface?